include/dfu/mcuboot.h

```diff
@@ -28,6 +28,15 @@
 /** Swap failed because image to be run is not valid */
 #define BOOT_SWAP_TYPE_FAIL     5
 
+#define BOOT_IMG_VER_STRLEN_MAX 25  /* 255.255.65535.4294967295\0 */
+
+struct image_version {
```

Comment:
can you rebase and align to newest master. - looks like this struct is already in the master version.
https://github.com/zephyrproject-rtos/zephyr/pull/5945/files#diff-c68b23d9ca27c62505b9ef202d7b49a0R23